src/odemis/util/img.py

```diff
@@ -432,6 +432,7 @@ def ensure2DImage(data):
     raise ValueError: if the data is not 2D (CTZ != 111)
     """
     d = data.view()
+
```

Comment:
Let's not go into minor aesthetic tweaks on random files. There are enough files already touched by this commit ;-) Please drop this change.

Reply:
Oops - I think i had added code here when I was trying to figure out what was wrong, and removed it later.